src/idl_gen_ts.cpp

```diff
@@ -406,6 +406,21 @@ class TsGenerator : public BaseGenerator {
       // return the bigint value directly since typescript does not support
       // enums with bigint backing types.
       switch (value.type.base_type) {
+        case BASE_TYPE_ARRAY: {
+          std::string ret = "[";
+          for (auto i = 0; i < value.type.fixed_length; ++i) {
+            ret +=
+                AddImport(imports, *value.type.enum_def, *value.type.enum_def)
```

Comment:
Save this to a local variable for readability

Reply:
done c716af815b3b6b5dd3017090a5bdd0ca434d4470